mod-source-record-storage-server/src/test/java/org/folio/rest/util/QueryParamUtilTest.java

```diff
@@ -75,4 +76,34 @@ public void shouldThrowBadRequestExceptionForUnknownRecordType() {
     QueryParamUtil.toRecordType("UNKNOWN");
   }
 
+  @Test
```

Comment:
maybe you can do it with @Parameterized test

Reply:
There could be a problem because this module is using junit4, which as I remember doesn't have an easy solution for parameterized tests.